doc/source/management/ssl-improvement.rst

```diff
@@ -0,0 +1,100 @@
+.. _percona-server.management.ssl:
+
+================================================================================
+SSL Improvements
+================================================================================
+
+|Percona Server| enables transmitting data in the encrypted form by using the
+TLSv1.2 protocol. By default, |Percona Server| disables TLSv1.0 and deprecates
+TLSv1.1.
+
+.. rubric:: Implemented Using |openssl|
+
+|Percona Server| uses the |openssl| toolkit version 1.1 to implement the suport
+of TLSv1.2 protocol. As part of its implementation, |Percona Server| offers
+correct diagnostic messages.
+
+.. seealso::
+
+   |MySQL| Bug System (solved for |Percona Server|): 
```

Comment:
```suggestion
   |MySQL| bug (solved for |Percona Server|): 
```

Reply:
while it is ok for the case when a reference is made to a single MySQL bug, it will have to be changed to something like MySQL bugs when two or more bugs are listed. Having to change this placeholder text (it's purpose is draw the readers' attention and not to provide valuable information) can become another source of spelling errors. 
Therefore, i'd suggest that this text remain unchanged (this is how MySQL defines it anyway)